ro-base.obo

```diff
@@ -2496,6 +2509,7 @@ property_value: IAO:0000117 https://orcid.org/0000-0002-6601-2165
 property_value: IAO:0000118 "actively involved in" xsd:string
 property_value: IAO:0000118 "enables part of" xsd:string
 property_value: seeAlso Involved:in
+property_value: seeAlso Involved:in
```

Comment:
???

Reply:
Something always didn't work with the URL `http://wiki.geneontology.org/` in OBO. The line 2511 is there since a release in 2018. In the `ro-base.owl` we have
`<rdfs:seeAlso rdf:resource="http://wiki.geneontology.org/index.php/Involved_in"/>`

Reply:
Maybe because of the underscore?

Reply:
Fixed in the commit [c0ecc1](https://github.com/oborel/obo-relations/pull/728/commits/c0ecc1df96a957bd979bc98214b85b855e6818ad)